fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/importhandler/loanrepayment/LoanRepaymentImportHandler.java

```diff
@@ -126,6 +126,8 @@ public Count importEntity(String dateFormat) {
 
                 JsonObject loanRepaymentJsonob = gsonBuilder.create().toJsonTree(loanRepayment).getAsJsonObject();
                 loanRepaymentJsonob.remove("manuallyReversed");
+                loanRepaymentJsonob.addProperty("glAccountId", 1);
```

Comment:
GL Account 1 will be debited when a repayment is made
In order to debit the product GL account, leave this blank and handle the error in the repayment processor

Reply:
@fiter-julius-oketayot Please check now